code/controllers/subsystem/throwing.dm

```diff
@@ -85,9 +87,9 @@ SUBSYSTEM_DEF(throwing)
 		delayed_time += world.time - last_move
 		return
 
-	if (dist_travelled && hitcheck()) //to catch sneaky things moving on our tile while we slept
-		finalize()
-		return
+	// if (dist_travelled && hitcheck()) //to catch sneaky things moving on our tile while we slept
```

Comment:
get_turf in hitcheck proc is causing thrown items to hit grilles underneath windows because the turf of the thrown item and the grille are the same and throws pass through windows because of the ON_BORDER_1 flag